scalatest/src/main/scala/org/scalatest/events/Event.scala

```diff
@@ -151,7 +151,7 @@ sealed abstract class Event extends Ordered[Event] with Product with Serializabl
           <depth>{ getThrowableStackDepth(throwable) }</depth>
           <stackTraces>
             {
-              val stackTraces = throwable.getStackTrace
+              val stackTraces = throwable.getStackTrace.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.